packages/server/tests/acceptance.spec.ts

```diff
@@ -320,6 +344,37 @@ describe('RPC Server Integration Tests', async function () {
         callUnsupportedRelayMethod(this.relayClient, 'eth_protocolVersion', []);
     });
 
+    it('erc20 test', async function () {
```

Comment:
I think that we should move away from the concept of writing our own implementations for calling contracts, creating them etc.

If possible (contract is <6kb) we should do everything inside the acceptance tests as if they were executed from a client dapp script. Example for that would be the following:


```TypeScript
const provider = new ethers.provider.JsonRpcProvider('http://localhost:8545');
const wallet =  new ethers.Wallet(evmCompPK, provider);

// deploy the contract through Hedera SDK since it is > 6kb

const erc20 = new ethers.Contract('0 prefixed address', erc20Abi, wallet);
// now you can do things like
const approveTx = await erc20.approve('some-account-0-prefixed', 100000);

```